sdks/python/apache_beam/io/filesystem.py

```diff
@@ -679,7 +679,7 @@ def match(self, patterns, limits=None):
 
     def _match(pattern, limit):
       """Find all matching paths to the pattern provided."""
-      if pattern.endswith('/'):
+      if pattern.endswith('/') or pattern.endswith('\\'):
```

Comment:
Do we other places that needs this change?

Reply:
I didn't find any other checks for `'/'`.